tasks/bump.js

```diff
@@ -143,7 +144,7 @@ module.exports = function(grunt) {
     runIf(opts.commit, function() {
       var commitMessage = opts.commitMessage.replace('%VERSION%', globalVersion);
 
-      exec('git commit ' + opts.commitFiles.join(' ') + ' -m "' + commitMessage + '"', function(err, stdout, stderr) {
+      exec('git commit ' + opts.commitFiles.join(' ') + (opts.skipPrecommitHook ? ' -n ' : '') + ' -m "' + commitMessage + '"', function(err, stdout, stderr) {
```

Comment:
move to a variable use that variable in the line as it is getting rather long. Also, use the full `--no-verify` flag so its more clear what it is doing